.github/workflows/build-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Build, Publish Release
+on:
+  release:
+    types: [released]
+
+jobs:
+  build-and-publish-release:
+    env:
+      GRGIT_USER: GitHub
+      GRGIT_PASS: ${{ secrets.GITHUB_TOKEN }}
+      GRADLE_ARGS: -Partifactory_user=${{secrets.ARTIFACTORY_USER}} -Partifactory_password=${{secrets.ARTIFACTORY_TOKEN}}
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 17
+
+      - name: Login to Artifactory
+        run: |
+          docker login -u ${{secrets.ARTIFACTORY_USER}} -p ${{secrets.ARTIFACTORY_TOKEN}} armory-docker-local.jfrog.io
+        shell: bash
+
+      - name: Publish To Artifactory
```

Comment:
I think this one is pushing the docker image 
`  ./gradlew --no-build-cache build -x check dockerPush $GRADLE_ARGS`

also it has references to `armory-enterprise-console` which should not happen

Reply:
removing all docker related actions